bitnami/external-dns/values-production.yaml

```diff
@@ -300,6 +300,7 @@ rfc2136:
   tsigSecretAlg: hmac-sha256
   tsigKeyname: externaldns-key
   tsigAxfr: true
+  minTTL: "0s"
```

Comment:
I would say the unit is optional [here](https://github.com/kubernetes-sigs/external-dns/blob/master/docs/ttl.md#configure-dns-record-ttl-time-to-live) and it is seconds by default, so you can avoid it. Also, I would add some docs about `time.Duration` and `time.ParseDuration` such as possible values.

```yaml
  ## Default unit `s`. Possible values [ns, us, ms, s, m, h], see more https://golang.org/pkg/time/#ParseDuration
  minTTL: "0"
```

Reply:
Initially, I implemented it without the unit, but errors show up. I will give it a try and post here the results. About the docs, that's a nice idea!

Reply:
This is the outcome you get when no units is provided (using a magnitude of 5):
```console
time="2020-05-27T08:11:10Z" level=fatal msg="flag parsing error: time: missing unit in duration 5"
```

Reply:
Cool, it seems they have implemented a validation in their side wrapping the original library, so let's keep the unit!